src/main/java/com/ericsson/ei/services/SubscriptionService.java

```diff
@@ -207,4 +220,27 @@ private String generateQuery(String subscriptionName, String ldapUserName) {
         }
```

Comment:
In this functiuion you may edit the if-statement like this.

boolean ownerExist = doSubscriptionOwnerExist(subscriptionName);
if (!ldapUserName.isEmpty() && ownerExist) {
}

Reply:
I rephrased it